vizro-core/docs/pages/user-guides/deploy.md

```diff
@@ -0,0 +1,195 @@
+
+    - If you are the only user of your app then the process is often simpler since you might never want to deploy to production.
+    - If there are multiple people involved with development then you will need some way to coordinate code changes, such as a [GitHub repository](https://github.com/) or [Hugging Face Space](https://huggingface.co/spaces).
+    - You might want to make your app accessible to only some people using _authentication_.
+    - You might want to update your dashboard after it has been put into production. There is then a cycle of repeated development and deployment.
+    - There might be additional stages or _environments_ for Quality Assurance (QA) to test that the app works correctly before it is deployed.
+
+## Overview
+
+When developing your dashboard you typically run it _locally_ (on your own computer) using the Flask development server. When you deploy to production, this is no longer suitable. Instead, you need a solution that can handle multiple users in a stable, secure and efficient way.
+
+Vizro is designed to be a production-ready framework, which means that the dashboard you created during development is immediately suitable for deployment to production with minimal changes. Under the hood, Vizro benefits from [Dash's stateless architecture](https://dash.plotly.com/sharing-data-between-callbacks#dash-is-stateless), which makes it simple to scale your app to thousands of concurrent users.
+
+In general, there is only one code change that is required. The last of your `app.py` file should look like `Vizro().build(dashboard).run()`. This should be replaced with:
+
+```python
+app = Vizro().build(dashboard)  # (1)!
+
+if __name__ == "__main__":  # (2)!
+    app.run()
+```
+
+1. The Vizro `app` object needs to be exposed so that the app can be started in production.
+1. Enable the same app to be [run in development](run.md#development-server) with the Flask development server. This code is only executed when you run `python app.py` and does not run in production.
+
+That's it! Your app is now suitable for deployment to production.
+
+!!! warning "Extra step if you use dynamic data cache"
+    If your dashboard uses [dynamic data](data.md#dynamic-data) that can be refreshed while the dashboard is running then you should [configure your data manager cache](data.md#configure-cache) to use a backend that supports multiple processes.
+
+Now that your `app.py` file is ready, you need to choose a _hosting provider_. There are many services out there with different offerings, but for most users we recommend two in particular: [Hugging Face](https://huggingface.co/) and [Ploomber Cloud](https://docs.cloud.ploomber.io/). These both have a free tier with the possibility of paying more for extras, and they are both quick and easy to get started with. We give step-by-step instructions on how to use each:
+
+- [Instructions for Hugging Face](#hugging-face)
```

Comment:
```suggestion
- [Deploy a Vizro dashboard to Hugging Face](#hugging-face)
```

Reply:
I've changed the link text as it'll be what people search Google to find out, so this makes it a bit easier for indexing and thus for them to find it.